modules/oe_content_publication/config/post_updates/00004_create_fields/field.field.node.oe_publication.body.yml

```diff
@@ -0,0 +1,24 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.body
+    - node.type.oe_publication
+  module:
+    - text
+_core:
+  default_config_hash: epQrM0F0rqe-Y_2pNLD5jzqSV_ZAWtwN2rfls_xQIno
```

Comment:
@georgebaev on a second thought, please remove all of these hash and let's create them in the update hook, keeping the comment. We are afraid we will forget the reason why we do this in a few months time and, even if we won't, this would look unusual to new developers, so better having it documented in the update hook.

Reply:
I applied the changes.